scripts/plugins/extauth-hook-AD.py

```diff
@@ -233,6 +223,69 @@ def _install(self):
                 os.remove(self._file_path)
 
 
+class UsersList(DynamicPam):
+    #pylint: disable=too-few-public-methods
+    """Class manage users which permit pool admin ssh"""
+
+    def __init__(self, session, arg, ad_enabled=True):
+        super(UsersList, self).__init__(HCP_USERS, session, arg, ad_enabled)
+
+    def _match_subject(self, subject_rec):
+        return subject_rec["other_config"]["subject-is-group"] != "true"
```

Comment:
What if `subject-is-group` is missing from `other-config`?

Reply:
Also why `other_config` instead of `other-config`, are you sure it is with `_`? Where is this set? Might be useful to use a string constant for other_config to ensure consistent spelling.

Reply:
`subject-is-group` comes from xapi database, and it will always have a value, 
As it always better to do security check, I will add a try catch block for this.